### PR TITLE
add second argument to allow for custom org

### PR DESCRIPTION
**Pull request by MylesBorins**
this will allow us to generate notes for other orgs than nodejs e.g. openjs-foundation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,18 @@ $ node-meeting-agenda lts-agenda
...
+```text
+$ node-meeting-agenda bootstrap-agenda openjs-foundation # optional second argument for custom org
+### openjs-foundation/bootstrap
+
+* Infrastructure: Mailing lists [#134](https://github.com/openjs-foundation/bootstrap/issues/134)
+* add PROJECT.md file [#132](https://github.com/openjs-foundation/bootstrap/pull/132)
+* Create process for new projects joining foundation [#130](https://github.com/openjs-foundation/bootstrap/issues/130)
+* Review meeting times [#129](https://github.com/openjs-foundation/bootstrap/issues/129)
+* Project Housekeeping: Create Checklist for New Projects / Updating Existing Projects [#121](https://github.com/openjs-foundation/bootstrap/issues/121)
+* Steps to bootstrap CPC and new Foundation [#115](https://github.com/openjs-foundation/bootstrap/issues/115)
```
Comment (mhdawson) [inline README.md:36]: This comment seems to have some additional text which was not intended ?

Comment (MylesBorins) [reply README.md:36]: If you look above there are examples of other similar calls to `node-meeting-agenda` and the results

Comment (mhdawson) [reply README.md:36]: ah ok see it is an example now :)

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
@rvagg plan to land tomorrow unless you object.

**Comment by rvagg** [comment]
please don't consider me a gate for this, if you need anything from me to make it happen then let me know, otherwise here's a blanket 👍

**Pull request merged**